tests/instruments/rohdeschwarz/test_fsl.py

```diff
@@ -0,0 +1,79 @@
+# from time import sleep
+
+import pytest
+from pymeasure.instruments.rohdeschwarz.fsl import FSL
+
+# pytest.skip("Only works with connected hardware", allow_module_level=True)
+RESOURCE = "TCPIP::141.20.43.231::INSTR"
+
+
+@pytest.fixture
+def fsl_default_state():
+    """
+    Return an FSL instrument.
```

Comment:
...in its default state

Reply:
Resolved via [97b52bd](https://github.com/bleykauf/pymeasure/pull/4/commits/97b52bdbb7720ee70b29b289be5a6ca617880581)